### ISSUE TITLE
Make denovogear pass sanity check

### ISSUE THREAD

**Issue by JesperBrink** [Hogfeldt/bioconda_recipe_gen#89]

**Comment by JesperBrink** [comment]
This package was on the list of packages with a CMakeLists.txt that could pass the sanity check with its own recipe. 
But for some reason it doesn't pass the sanity check with its own recipe when I build it manually. 

A reason could be that it was in the 'build-fail-blacklist'  file (created by BIOCONDA-utils), which I had to manually remove it from before it would build. Before I removed it from the blacklist the output was:
```
07:44:38 BIOCONDA INFO Considering total of 1 recipes (denovogear).
07:44:38 BIOCONDA INFO Ignoring 1 blacklisted recipes (denovogear).
07:44:38 BIOCONDA INFO Processing 0 recipes.
07:44:38 BIOCONDA INFO Nothing to be done.
```
This might have given an error code of 0, which is what we check for to see if it was a successful built.
